### PR TITLE
Product order and image integration

### PR DESCRIPTION
**Pull request by mirafedas**
Two components: [product order](https://prnt.sc/pltxu9) and [product image](https://prnt.sc/pltxms) have been integrated into PDP.

### CODE REVIEW

**Comment by Hollister009** [comment]
@KhrystynaVaskovych can you review this one for me?

**Comment by KhrystynaVaskovych** [comment]
@Hollister009 , sure!

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import './Button.scss';
```
Comment (KhrystynaVaskovych) [inline src/components/Button/Button.js:4]: empty line would be great before scss import :)

File: src/components/ProductDetailsPage/ProductDetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { defaultImages } from '../../constants';
+import { productOrderParameters } from '../../config/ProductOrderMockups';
+
+import './ProductDetailsPage.scss';
+
+export const CN = 'product-details';
+
+const ProductDetailsPage = () => (
+  <div className={`${CN} content`}>
+    <ProductImage imagesArr={defaultImages} />
```
Comment (KhrystynaVaskovych) [inline src/components/ProductDetailsPage/ProductDetailsPage.js]: it's good to name variables without 'Arr'

File: src/components/ProductDetailsPage/ProductDetailsPage.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import "../../scss/global_styles";
```
Comment (KhrystynaVaskovych) [inline src/components/ProductDetailsPage/ProductDetailsPage.scss]: import with single quotes)

File: src/components/ProductImage/ProductImage.js [hunk 4]
```diff
@@ -31,6 +25,14 @@ class ProductImage extends Component {
...
+  clickHandler = ({ target }) => {
+    const selectedImage = target.dataset.src;
+
+    this.setState({
+      selectedImage
+    });
+  }
+
   renderSmallImages(selectedImage) {
     const { imagesArr } = this.props;
```
Comment (KhrystynaVaskovych) [inline src/components/ProductImage/ProductImage.js]: same here^ would be better without 'Arr'

File: src/components/ProductImage/ProductImage.scss [hunk 5]
```diff
@@ -1,40 +1,41 @@
 @import "../../scss/global_styles";
```
Comment (KhrystynaVaskovych) [inline src/components/ProductImage/ProductImage.scss]: trouble with quotes, should be single

File: src/components/ProductImage/ProductImage.scss [hunk 6]
```diff
@@ -43,17 +44,26 @@
     bottom: 0;
     left: 0;
     right: 0;
+    width: 100%;
+    background-color: rgba(244, 162, 54, 0.7);
```
Comment (KhrystynaVaskovych) [inline src/components/ProductImage/ProductImage.scss]: would be good to create a variable:)

File: src/components/ProductImage/ProductImage.scss [hunk 7]
```diff
@@ -43,17 +44,26 @@
...
+    outline: none;
   
     &:hover {
       opacity: 1;
     }
   }
 
   &__selected-image {
-    border: 5px solid rgba(244, 162, 54, 1);;
+    border: 2px solid rgba(244, 162, 54, 1);;
```
Comment (KhrystynaVaskovych) [inline src/components/ProductImage/ProductImage.scss]: same here, be so kind to add to variables, perhaps we'll need it)

File: src/components/Button/Button.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+@import "../../scss/global_styles";
```
Comment (KhrystynaVaskovych) [inline src/components/Button/Button.scss]: please, change to single quotes:)

File: src/components/ProductDetailsPage/ProductDetailsPageContainer.js [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  render() {
+    const { product } = this.state;
+    const productName = product.name;
+    const productId = product.id;
+
+    return (
+      <div>
+        <div>{productName}</div>
```
Comment (KhrystynaVaskovych) [inline src/components/ProductDetailsPage/ProductDetailsPageContainer.js]: why div?

**Review by KhrystynaVaskovych — CHANGES_REQUESTED** [review]
Well done! :) Needs miiiinor corrections)

**Review by Nazar-bnm — APPROVED** [review]

**Pull request merged**